### PR TITLE
docs(components): Add design and usage guidelines to tooltip

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivation

Make it easier to know when and why to use tooltips

## Changes

There are design & usage guidelines now

### Added

There are design & usage guidelines now

### Changed

tweaked intro copy

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Tooltip/Tooltip.mdx [hunk 1]
```diff
@@ -32,6 +32,19 @@ import { Tooltip } from "@jobber/components/Tooltip";
...
 </Playground>
 
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Tooltip component, consider the following:
+
+### Usage
+- A tooltip should be used to introduce a novel UI element or highlight a change to an existing interface.
+- A tooltip should contain information that is helpful, but not necessary for the user to successfully interact with the page.
+- A tooltip should not be a permanent fixture in the interface - if you need a message that remains in the view, consider putting that content inline.
```
Comment (agglasaur) [inline packages/components/src/Tooltip/Tooltip.mdx:42]: This last one I'm unclear on. What do you mean a permanent fixture? A lot of tooltips are permanent in the interface

File: packages/components/src/Tooltip/Tooltip.mdx [hunk 2]
```diff
@@ -32,6 +32,19 @@ import { Tooltip } from "@jobber/components/Tooltip";
...
+When contributing to, or consuming the Tooltip component, consider the following:
+
+### Usage
+- A tooltip should be used to introduce a novel UI element or highlight a change to an existing interface.
+- A tooltip should contain information that is helpful, but not necessary for the user to successfully interact with the page.
+- A tooltip should not be a permanent fixture in the interface - if you need a message that remains in the view, consider putting that content inline. 
+
+### Content
+- Tooltip content should consist of only text and an optional "dismiss" CTA
+- Tooltip text content should not exceed 3 lines
```
Comment (agglasaur) [inline packages/components/src/Tooltip/Tooltip.mdx:46]: ty for specifying length, we've got some long bois out there 🙏

File: packages/components/src/Tooltip/Tooltip.mdx (hunk 1 above)
Comment (chris-at-jobber) [reply packages/components/src/Tooltip/Tooltip.mdx:42]: I think that's more my ideal state going forward and less a reflection of current state... If something is hidden in a tooltip and we're just using that to save space it's probably not a great UI solution?

File: packages/components/src/Tooltip/Tooltip.mdx (hunk 2 above)
Comment (chris-at-jobber) [reply packages/components/src/Tooltip/Tooltip.mdx:46]: l o o o n g b o i s

File: packages/components/src/Tooltip/Tooltip.mdx (hunk 1 above)
Comment (agglasaur) [reply packages/components/src/Tooltip/Tooltip.mdx:42]: I think there may always be supplementary information that are just not necessarily a problem of the UI (see: pricing grid and its million tooltips). Sure there may be other ways to handle that, but I do think tooltips have their place with permanence in a UI

Comment (chris-at-jobber) [reply packages/components/src/Tooltip/Tooltip.mdx:42]: Yeah that's a fair point

There's also the idea of a tooltip as a bonus button label (which we don't do but could be kind of nice in the future) that systems like Material do pretty well

I think maybe the previous line about "helpful, but not necessary" covers what I was trying to say here

**Pull request merged**